tests/testthat/test-unnecessary_lambda_linter.R

```diff
@@ -13,7 +13,7 @@ test_that("unnecessary_lambda_linter skips allowed usages", {
   expect_lint("parLapply(cl, x, function(xi) foo(xi, 2, bar(baz(xi))))", NULL, linter)
 
   # multi-argument case
-  expect_lint("lapply(l, function(x, y) setdiff(x, y))", NULL, linter)
+  expect_lint("lapply(l, function(x, na.rm = TRUE) mean(x, na.rm = na.rm))", NULL, linter)
```

Comment:
I still don't see why you'd need a lambda here.

lapply(l, mean, na.rm=TRUE) does the same.

Reply:
Fair enough.